src/Presentation/HtmlPredicatesPresenter.php

```diff
@@ -10,36 +10,74 @@
 
 class HtmlPredicatesPresenter implements PredicatesPresenter {
 
+	private const ASTERISK = '<code>*</code>';
```

Comment:
If you wish to avoid the * being treated as wikitext, you can use `<nowiki></nowiki>`, which does not add additional markup

Reply:
Does the * need to look different or can it just be plain? If it doesn;t need to look special it will simplify some things.

Reply:
(I have no idea what actually looks better here. Just an FYI in case you do not know about that tag.)

Reply:
this is now:
![Screenshot_20220821_235955](https://user-images.githubusercontent.com/1428594/185812593-a2d3cc72-c8b2-48ac-8ab6-92a6fabaf96d.png)

or just plain:
![Screenshot_20220822_000027](https://user-images.githubusercontent.com/1428594/185812617-44111c5e-7c99-41be-8e81-474335343100.png)

Reply:
The first one looks kinda weird and the second one is not great either. But at the same time, either is good enough.